terraform/projects/app-ecs-albs/main.tf

```diff
@@ -158,7 +164,7 @@ resource "aws_route53_record" "prom_alias" {
 }
 
 resource "aws_route53_record" "alerts_alias" {
-  count = "${length(data.terraform_remote_state.infra_networking.public_subnets)}"
+  count = "${local.alerts_records_total}"
```

Comment:
this should be `local.alerts_records_count`, no?

Reply:
should be!